modules/contentbox/migrations/2022_10_14_173737_CONTENTBOX-1429-Slow-Admin-Search.cfc

```diff
@@ -0,0 +1,12 @@
+component {
```

Comment:
Can we modify the name to follow the convention mentioned before please.  Thanks!